providers/catalog.go

```diff
@@ -467,4 +468,23 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// MS Sales configuration
+	MicrosoftDynamics365Sales: {
+		AuthType: Oauth2,
+		BaseURL:  "https://{{.workspace}}.api.crm.dynamics.com/api/data/v9.2",
```

Comment:
Can you please remove the suffix in this? We want users of this to be able to access all versions. I would keep the base URL as just `https://{{.workspace}}.api.crm.dynamics.com` - there might be other sections of the API that might be accessible via this

Reply:
That makes sense. So when going over proxy the url would look like `http://localhost:444/api/data/v9.2` giving end user all control. I believe v9.2 uri part only changes in their API